### DISCUSSION TITLE
Import OFX and/or Quicken QIF format

### DISCUSSION THREAD

**Discussion by arthurlutz** [ananthakumaran/paisa#193] [Ideas]
It would be nice to be able to import OFX files from the interface.

**Comment by ananthakumaran** [comment]
Would you be able to share some sample files? (anonymized) OFX/QIF is unheard of in my country and I have found it difficult to get sample files.

**Reply by arthurlutz** [reply]
Hi, sorry for taking so long to respond. 

Here is a sample that looks very similar to what I'm downloading from the bank : https://gist.github.com/jvz/2837829 

Apparently https://github.com/wesabe/fixofx/blob/master/fakeofx.py can also generate some samples. 

I know that in the past I used https://skrooge.org/ that supports OFX and might have some samples there too (or export to OFX). And ledger 3 seems to have removed OFX so there might be some samples in the 2.x versions... 

I might try the CSV download from my bank with paisa but was looking for a way to import without too much mapping.
